src/edu.umn.cs.melt.copper.compiletime/src/main/java/edu/umn/cs/melt/copper/main/ParserCompiler.java

```diff
@@ -78,6 +78,8 @@ private static String usageMessage(ParserCompilerParameters args)
 		rv += "\t-mda\tRun Copper's modular determinism analysis on the input.\n\t\t\tIf this switch is used, the input must comprise exactly\n\t\t\ttwo grammars: the host and an extension to test.\n";
 		rv += "\t-avoidRecompile\tRun Copper only if one spec-file has a later\n\t\t\tmodification time than the output file.\n\t\t\tIf this switch is used, an output file must also\n\t\t\tbe specified.\n";
 		rv += "\t-logfile [lout]\tPipe all log output to the file 'lout'\n\t\t\t(default standard error).\n";
+		rv += "\t-dot [dout]\tOutput Graphiz dot code representing any counterexamples to 'dout'\n\t\t\t(no dot output if not provided).\n";
```

Comment:
The name of Graphviz is consistently misspelled across this PR. I would resolve this by removing the Graphviz references from the names altogether, since the DOT format is supported by more than one graph layout tool.

Reply:
Makes sense, fixed.